server/src/main/java/org/opensearch/cluster/routing/remote/RemoteRoutingTableService.java

```diff
@@ -61,6 +64,11 @@ protected void doStart() {
         blobStoreRepository = (BlobStoreRepository) repository;
     }
 
+    public void deleteStaleIndexRoutingPaths(List<String> stalePaths) throws IOException {
+        logger.debug(() -> "Deleting stale index routing files from remote - " + stalePaths);
+        blobStoreRepository.blobStore().blobContainer(BlobPath.cleanPath()).deleteBlobsIgnoringIfNotExists(stalePaths);
+    }
+
```

Comment:
Do we need to do exception handling?

Reply:
`deleteStaleIndexRoutingPaths` function is called from function `deleteClusterMetadata` that handles the necessary exceptions.